vsts/sdl.yaml

```diff
@@ -71,11 +71,11 @@ phases:
       codebase: NewOrUpdate
       tsaEnvironment: PROD
       codeBaseName: 'Azure-Iot-SDK-Java-Master'
-      notificationAlias: 'timtay@microsoft.com, prmathur@microsoft.com, jasminel@microsoft.com'
-      codeBaseAdmins: 'REDMOND\timtay;REDMOND\prmathur;REDMOND\jasminel'
+      notificationAlias: 'aziotdndmgdsdk@microsoft.com'
+      codeBaseAdmins: 'REDMOND\timtay;REDMOND\jamdavi;REDMOND\azabbasi'
       instanceUrlForTsaV2: MSAZURE
       projectNameMSAZURE: One
-      areaPath: 'One\IoT\Platform\Clients\java'
+      areaPath: 'One\IoT-Platform-Devices-Java'
```

Comment:
How do we know this is the area path? I'm confused about the updates.

Reply:
I talked with some of the people who maintain this TSA upload task, and they helped me find this info. I'll fwd you the email

Reply:
@wickste this is the area path that we'd like some confirmation on before we commit

Reply:
In case this helps, this is where we've been uploading the .NET logs for a while now: https://github.com/Azure/azure-iot-sdk-csharp/blob/master/vsts/vsts.yaml#L321-L333
```
projectNameMSAZURE: 'One'
areaPath: 'One\IoT\Developers and Devices\SDKs\Managed'
iterationPath: 'One\IoT\Backlog'
```

In case we get a different guidance, we should update the .NET path as well.